examples/model_ensemble_to_rcmed.py

```diff
@@ -125,6 +125,11 @@
     wrf311_dataset, temporal_resolution='annual')
 cru31_dataset = dsp.temporal_rebin(cru31_dataset, temporal_resolution='annual')
 
+# Rebinning annual pushes the start date to the middle of the year
```

Comment:
This example has not been updated for a while, but start_time = end_time does not make any sense. I will take a look at this example again.

Reply:
The example has 1 year of data.  1/1/1989 to 12/31/1989.

The data is then rebinned to annual.

That I believe that effectively pushes all the data to a single point a 7/2/1989.

I reset start date to the start date post rebinning.  Setting the end date to the start date was easiest, but I could have set it to the end date on the data set, but I believe that would have been 7/2/1989 too.

Reply:
The original script compares the annual means for the year 1989 between cru (observation), two models, and an average of the models. I think using the start and end times of (1989, 1, 1) and (1989, 12, 1) are totally okay and drawing three plots for the three model datasets is reasonable.